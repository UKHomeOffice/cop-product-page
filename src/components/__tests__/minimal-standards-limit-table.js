import React from "react";
import renderer from "react-test-renderer";
import MinimalStandardLimitsTable from "../minimal-standard-limits-table";

describe("MinimalStandardLimitsTable", () => {
  it("renders correctly in content", () => {
    const tree = renderer
      .create(
        <MinimalStandardLimitsTable
          table={[
            {
              column1: "Cyclamen",
              column2:
                "&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;",
              column3: "&nbsp; &nbsp",
            },
          ]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
