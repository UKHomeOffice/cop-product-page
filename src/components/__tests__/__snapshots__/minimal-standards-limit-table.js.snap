// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MinimalStandardLimitsTable renders correctly in content 1`] = `
<div>
  <div
    className="govuk-grid-row displayFlexOnDesktop"
  >
    <div
      className="govuk-grid-column-one-third minimalStandardsTable"
      style={
        Object {
          "backgroundColor": "#00703c",
          "margin": "1px",
        }
      }
    >
      <div
        className="govuk-body"
        dangerouslySetInnerHTML={
          Object {
            "__html": "Cyclamen",
          }
        }
        style={
          Object {
            "color": "white",
            "marginBottom": "auto",
            "marginTop": "auto",
            "padding": "10px",
            "textAlign": "left",
          }
        }
      />
    </div>
    <div
      className="govuk-grid-column-one-third minimalStandardsTable"
      style={
        Object {
          "backgroundColor": "rgba(0, 112, 60, 0.2)",
          "margin": "1px",
        }
      }
    >
      <div
        className="govuk-body"
        dangerouslySetInnerHTML={
          Object {
            "__html": "&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;",
          }
        }
        style={
          Object {
            "marginBottom": "auto",
            "marginTop": "auto",
            "padding": "10px",
            "textAlign": "left",
          }
        }
      />
    </div>
    <div
      className="govuk-grid-column-one-third "
      style={
        Object {
          "backgroundColor": "rgba(0, 112, 60, 0.2)",
          "margin": "1px",
        }
      }
    >
      <div
        className="govuk-body"
        dangerouslySetInnerHTML={
          Object {
            "__html": "&nbsp; &nbsp",
          }
        }
        style={
          Object {
            "marginBottom": "auto",
            "marginTop": "auto",
            "padding": "10px",
            "textAlign": "left",
          }
        }
      />
    </div>
  </div>
  <div
    style={
      Object {
        "marginTop": "20px",
      }
    }
  />
</div>
`;
